pkg/processor/deployment/deployment.go

```diff
@@ -142,6 +145,28 @@ func (d deployment) Process(appMeta helmify.AppMetadata, obj *unstructured.Unstr
 	}, nil
 }
 
+func addWebhookOption(manifest string) string {
+	webhookOptionHeader := "      {{- if .Values.webhook.enabled }}"
+	webhookOptionFooter := "      {{- end }}"
+	volumes := `      - name: cert
+        secret:
+          defaultMode: 420
+          secretName: webhook-server-cert`
+	volumeMounts := `        - mountPath: /tmp/k8s-webhook-server/serving-certs
+          name: cert
+          readOnly: true`
+	webhookPort := `        - containerPort: 9443
```

Comment:
is there a better way to handle webhooks? I've tried to search a couple of kubebuilder projects on GitHub and some of them are using [different ports](https://github.com/jetstack/kubebuilder-sample-controller/blob/master/config/default/manager_webhook_patch.yaml) or yaml properties can be ordered differently so `strings.ReplaceAll` will not work in most cases.

Reply:
Thanks for catching out, It looks like we could use the regular expression to handle these cases.

Reply:
@arttor Does such design makes sense to you?

Reply:
yes, regular expressions look promising.